Sources/SwiftFortuneWheel/Utils/Audio/AudioPlayerManager.swift

```diff
@@ -101,8 +95,15 @@ class AudioPlayerManager {
         
         func performPlaybackOnFirstAvailablePlayer() {
             guard let player = firstAvailablePlayer() else { return }
-            
-            player.play(audio, identifier: sound)
+            do {
+                engine.connect(player.node, to: engine.mainMixerNode, format: nil)
```

Comment:
connecting the player to the engine before playing the sound could cause sound delay.